test/metabase/api/dashboard_test.clj

```diff
@@ -1975,6 +2049,51 @@
                     :has_more_values false}
                    (mt/user-http-request :rasta :get 200 url)))))))))
 
+(deftest static-values-test
+  (mt/with-temp Dashboard [{dashboard-id :id} {:parameters [{:id             "abc"
+                                                             :type           "category"
+                                                             :name           "CATEGORY"
+                                                             :source_type    "static-list"
+                                                             :source_options {:values ["toucan" "pigeon" "other bird"]}}]}]
+    (testing "It uses static values stored directly in the parameters"
+      (let-url [url (chain-filter-values-url dashboard-id "abc")]
+        (is (= {:values          ["toucan" "pigeon" "other bird"]
+                :has_more_values false}
+               (mt/user-http-request :rasta :get 200 url)))))
+    (testing "it only returns search matches"
+      (let-url [url (chain-filter-search-url dashboard-id "abc" "OUC")]
+        (is (= {:values          ["toucan"]
+                :has_more_values false}
+               (mt/user-http-request :rasta :get 200 url)))))))
+
+(deftest parameter-values-from-card-test
+  (mt/with-temp* [Card          [{card-id :id}      {:database_id   (mt/id)
+                                                     :table_id      (mt/id :venues)
+                                                     :dataset_query (mt/native-query {:query "SELECT name FROM venues WHERE name LIKE '%Bar%'"})}]
+                  Dashboard     [{dashboard-id :id} {:parameters [{:id             "abc"
+                                                                   :type           "category"
+                                                                   :name           "CATEGORY"
+                                                                   :source_type    "card"
+                                                                   :source_options {:card_id card-id}}]}]
+                  ParameterCard [_                  {:card_id                   card-id
```

Comment:
I'm confused about why this isn't getting created automatically. It's basically the exact same setup in this test below:

https://github.com/metabase/metabase/pull/27008/files#diff-fc6fec87516cb4792942d9d74016b2ce2aa80fbd72c34c963003cd0918b86ae4R219